.github/workflows/regression-report.yml

```diff
@@ -0,0 +1,248 @@
+#####################################################################################
+# GitHub Action to generate Checkstyle report.
+#
+# Workflow starts when:
+# 1) issue comment - created
+#
+# Requirements:
+# 1) secrets.AWS_ACCESS_KEY_ID - access key for AWS S3 service user
+# 2) secrets.AWS_SECRET_ACCESS_KEY - security access key for AWS S3 service user
+#
+# If you need to change bucket name or region, change AWS_REGION and AWS_BUCKET_NAME variables.
+# For another bucket, you will need to change the secrets.
+#####################################################################################
+name: regression-report
+env:
+  AWS_REGION: us-east-2
+  AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  USER_LOGIN: ${{ github.event.issue.user.login }}
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+concurrency:
+  # run_id is guaranteed to be unique and present
+  group: ${{ github.run_id }}
+  cancel-in-progress: false
+
+jobs:
+  parse_comment:
+    if: startsWith(github.event.comment.body, 'Github, generate report for')
+    runs-on: ubuntu-latest
+    outputs:
+      config_bundle_path: ${{ steps.parse.outputs.config_bundle_path }}
+      config_link: ${{ steps.set_links.outputs.config_link }}
+      projects_link: ${{ steps.set_links.outputs.projects_link }}
+      branch: ${{ steps.set_branch.outputs.ref }}
+      commit_sha: ${{ steps.set_branch.outputs.commit_sha }}
+
+    steps:
+      - uses: shanegenschaw/pull-request-comment-trigger@v2.1.0
+        name: React with rocket on run
+        with:
+          trigger: ','
+          reaction: rocket
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+      - run: 'echo We print it here for this action to work'
+        if: 'true'
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Parse comment
+        id: parse
+        env:
+          COMMENT_BODY: ${{ github.event.comment.body }}
+        run: |
+          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
+            grep -oP '(?<=Github, generate report for ).*')
```

Comment:
`$(echo "$COMMENT_BODY" | sed -E 's/.*, generate report for //'`

it replace phrase to empty string.

Reply:
Done